apps/web/components/real.ts

```diff
@@ -65,3 +65,4 @@ export * from './InstitutionPanel/InstitutionPanel'
 export * from './InstitutionsPanel/InstitutionsPanel'
 export * from './Organization/'
 export * from './ApiCatalogueFilter/ApiCatalogueFilter'
+export * from './Image'
```

Comment:
Looks like the component is never imported in the web project, not sure if it has been deprecated in favor of the contentful Image component or just not used anymore but will be in the future

Reply:
FWIW I'm using it on a local feature branch, ATM.
I can downgrade to a plain `<img/>` if needed, though.